### PR TITLE
release/public-v1:  Add FAQ chapter with the following questions:

### PR DESCRIPTION
**Pull request by JulieSchramm**
- How do I build and run a single test of the UFS Weather Model?
   - How do I change the length of the model run?
   - How do I change the model output to netcdf or nemsio?

- Add FAQ chapter to index.rst
- Add references to sections in other chapters
- Add FAQ chapter to introduction

Both html and latex builds succeed.

### CODE REVIEW

**Comment by llpcarson** [comment]
Good info to add, Julie!

File: doc/UsersGuide/source/FAQ.rst [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+==============================================================
+How do I build and run a single test of the UFS Weather Model?
+==============================================================
+
+An efficient way to build and run the UFS Weather Model is to use the regression test
+(``rt.sh``).  This script is widely used by model developers on Tier 1 and 2 platforms
+and is described in :numref:`Section %s <ConductingRegTests>`.  The advantages to this approach are:
+
+- You can bypass the workflow, pre- and post-processing steps.
```
Comment (ligiabernardet) [inline doc/UsersGuide/source/FAQ.rst]: I suggest changing the language here. The UFS WM does not have a workflow with pre- and post- that needs to be bypassed (the App does, but this documentation is for the model).
```suggestion
- It does not require a workflow, pre- or post-processing steps.
```

File: doc/UsersGuide/source/FAQ.rst [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+       qsub job_card
+
+============================================
+How do I change the length of the model run?
+============================================
+In your run directory, there is a file named ``model_configure``.  Change the
+variable ``nhours_fcst`` to the desired number of hours.
+
+=====================================================
+How do I change the model output to netcdf or nemsio?
```
Comment (ligiabernardet) [inline doc/UsersGuide/source/FAQ.rst]:
```suggestion
How do I select the file format for the model output (NetCDF or NEMSIO)?
```

**Review by ligiabernardet — CHANGES_REQUESTED** [review]

**Review by ligiabernardet — APPROVED** [review]
Looks good to me. Thanks.

**Review by llpcarson — APPROVED** [review]
approved

**Comment by ligiabernardet** [comment]
@junwang-noaa What is the process for having these documentation updates merged into the release branch? Note that we have a few more updates coming.

**Comment by junwang-noaa** [comment]
> @junwang-noaa What is the process for having these documentation updates merged into the release branch? Note that we have a few more updates coming.

Please submit PR to the release branch and it will be committed after reviewers' approval.

**Comment by ligiabernardet** [comment]
Thanks @junwang-noaa The PR has been submitted and the assigned reviewers have approved it. If necessary, you can assign additional reviewers. Thanks.

**Pull request merged**